File: iOS/PanoAudioChat/PanoAudioChatSDK/Source/Base/PanoBaseView.m
//
//  PanoBaseView.m
//  PanoVideoCall
//
//  Created by pano on 2020/9/28.
//  Copyright © 2020 Pano. All rights reserved.
//

#import "PanoBaseView.h"

@implementation PanoBaseView

- (instancetype)init {
    return [self initWithFrame:CGRectZero];
}

- (instancetype)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        [self initViews];
        [self initConstraints];
    }
    return self;
}

- (void)initViews {
    
}

- (void)initConstraints {
    
}

@end

File: iOS/PanoAudioChat/PanoAudioChat/Classes/Sections/PanoBaseView.swift
//
//  PanoBaseView.swift
//  PanoAudioChat
//
//  Created by wenhua.zhang on 2020/10/29.
//

import UIKit
import SnapKit
import UIColor_Hex_Swift
import Localize_Swift

class PanoBaseView: UIView {
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        initViews()
        initConstraints()
    }
    
    func initViews() {
        
    }
    
    func initConstraints() {
        
    }
    
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
    deinit {
        print(self, "deinit")
    }
    
}
